package.json

```diff
@@ -49,6 +49,7 @@
     "cross-spawn": "7.0.1",
     "crypto-js": "^3.1.9-1",
     "graphql": "14.4.2",
+    "https-proxy-agent": ">=3.0.0",
```

Comment:
what does this have to do with insecure mode? maybe remove this?

Reply:
Not sure why this has been end here :-/, but this module it's not currently used in develop so it's safe to remove it.